src/WorldStorage/FastNBT.cpp

```diff
@@ -30,85 +30,78 @@ static const int MAX_LIST_ITEMS = 10000;
 	#define PROPAGATE_ERROR(X) do { auto Err = (X); if (Err != eNBTParseError::npSuccess) return Err; } while (false)
 #endif
 
+#define AS_INT(x) static_cast<int>(x)
+
```

Comment:
Is this really necessary? To me this just adds confusion to what the code does later.

Reply:
I just though that it was too visually noisy with the full `static_cast` for every label:

```cpp
case static_cast<int>(eNBTParseError::npNoTopLevelCompound):
```